### PR TITLE
Add superset to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
@CurtLH, give me a thumbs up if you're still okay being a co-maintainer.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/superset/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    - bleach
+    - boto3 ==1.4.7
+    - botocore >=1.7.0,<1.8.0
+    - celery >=4.2.0
+    - click ==6.7
+    - colorama ==0.3.9
+    - contextlib2 #==0.5.5
+    - cryptography #==1.9
+    - flask <1.0.0
+    # Use fabmanager until 1.13, after that use flask fab
```
Comment (sodre) [inline recipes/superset/meta.yaml]:
```suggestion
```

File: recipes/superset/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  entry_points:
+    - superset = superset.bin.superset:cli
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+    # All dependencies come from github
```
Comment (sodre) [inline recipes/superset/meta.yaml]:
```suggestion
    # All dependency comments come from github
```

File: recipes/superset/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    - flask-wtf #==0.14.2
+    - flower #==0.9.2  # deprecated
+    - future >=0.16.0,<0.17
+    - geopy #==1.11.0
+    - gunicorn #==19.8.0  # deprecated
+    - humanize #==0.5.1
+    - idna #==2.6
+    - isodate #==0.6.0
+    - markdown >=3.0
+    # see apache/incubator-superset#
```
Comment (sodre) [inline recipes/superset/meta.yaml]:
```suggestion
```

**Comment by sodre** [comment]
@conda-forge/help-python the superset recipe is ready for review.

File: recipes/superset/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+    # All dependency comments come from github
+    # https://raw.githubusercontent.com/apache/incubator-superset/0.28/requirements.txt
+    - bleach
+    - boto3 ==1.4.7
```
Comment (mariusvniekerk) [inline recipes/superset/meta.yaml]: This is an extremely old version of boto3.  That might cause some issues...

**Comment by mariusvniekerk** [comment]
Thanks for this effort!  Kind of strange that there have been numerous other rc releases since this one but none of them seem to have been officially blessed.

Should we build those as well?  It's not really standard conda-forge policy but it seems that they are having issues with apache incubation and odds are that our users may be better served by having the RC releases available and easy to install.

File: recipes/superset/meta.yaml (hunk 4 above)
Comment (sodre) [reply recipes/superset/meta.yaml]: @mariusvniekerk, that is the version that they had in their setup.py. I can relax the constraint, but I have no easy way to add test to meta.yaml and make sure other things did not break. I'll defer to your judgement. Should I change it to >=1.4.7,<2 and trust AWS' semver?

**Comment by sodre** [comment]
@mariusvniekerk, I am happy to add the rc versions under the feedstock if I am able to build it. Should I maintain it under a separate branch?

**Comment by sodre** [comment]
I'll add that there is still value in building 0.28.1 because the version in PyPi is broken. I.e. a plain pip install of superset does not work out-of-the-box without downgrading Pandas and SQLAlchemy.  Our conda-package would not have this issue at the moment.

**Comment by mariusvniekerk** [comment]
merged!  I think its sensible to have an rc branch

**Pull request merged**